.github/workflows/tests.yml

```diff
@@ -119,7 +119,20 @@ jobs:
 
       - name: Start Covalent dispatcher server
         run: covalent start
-
+    
+#      - uses: dorny/paths-filter@v2
+#        id: filter
+#        with:
+#          filters: |
+#            sdk:
+#              - 'covalent/'
+#            dispatcher:
+#              - 'covalent_dispatcher/'
```

Comment:
can the comment blocks be removed if not needed?

Reply:
I'll clean up the comments after last commit. Thanks!